tests/test_templatetags.py

```diff
@@ -22,6 +22,99 @@ def test_add_query_param_with_non_latin_character(self):
         self.assertIn("q=%E6%9F%A5%E8%AF%A2", json_url)
         self.assertIn("format=json", json_url)
 
+    def test_format_value_boolean_or_none(self):
+        """
+        Tests format_value with booleans and None
+        """
+        self.assertEqual(format_value(True), '<code>true</code>')
+        self.assertEqual(format_value(False), '<code>false</code>')
+        self.assertEqual(format_value(None), '<code>null</code>')
+
+    def test_format_value_list(self):
+        """
+        Tests format_value with a list of strings
+        """
+        list_items = ['item1', 'item2', 'item3']
+        self.assertEqual(format_value(list_items), '\n item1, item2, item3\n')
+        self.assertEqual(format_value([]), '\n\n')
+
+    def test_format_value_table(self):
+        """
+        Tests format_value with a list of lists/dicts
+        """
+        list_of_lists = [['list1'], ['list2'], ['list3']]
+        self.assertEqual(format_value(list_of_lists), '\n<table class="table table-striped">\n  <tbody>\n    \n      <tr>\n        <th>0</th>\n        <td>\n list1\n</td>\n      </tr>\n    \n      <tr>\n        <th>1</th>\n        <td>\n list2\n</td>\n      </tr>\n    \n      <tr>\n        <th>2</th>\n        <td>\n list3\n</td>\n      </tr>\n    \n  </tbody>\n</table>\n')
+
+        list_of_dicts = [{'item1': 'value1'}, {'item2': 'value2'}, {'item3': 'value3'}]
+        self.assertEqual(format_value(list_of_dicts), '\n<table class="table table-striped">\n  <tbody>\n    \n      <tr>\n        <th>0</th>\n        <td></td>\n      </tr>\n    \n      <tr>\n        <th>1</th>\n        <td></td>\n      </tr>\n    \n      <tr>\n        <th>2</th>\n        <td></td>\n      </tr>\n    \n  </tbody>\n</table>\n')
+
+    def test_format_value_dict(self):
+        """
+        Tests format_value with a dict of strings
+        """
+        dict_items = {'item1': 'value1', 'item2': 'value2', 'item3': 'value3'}
+        self.assertEqual(format_value(dict_items), '')
```

Comment:
this seemed odd to me. The [template](https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/templates/rest_framework/admin/dict_value.html) was blank? Is this supposed to be blank?